cmake/SpectreAddTestLibs.cmake

```diff
@@ -121,9 +121,7 @@ function(add_test_library LIBRARY FOLDER LIBRARY_SOURCES LINK_LIBS)
     PRIVATE
     ${LINK_LIBS}
     SpectreFlags
-    Python::NumPy
-    Python::Python
-    Python::SciPy
+    Framework
```

Comment:
Shouldn't this target be called `TestingFramework`? Not for this PR of course

Reply:
:man_shrugging: probably, you're welcome to bring it up at the spectre call and if people agree submit a PR :)